.github/workflows/dart.yml

```
@@ -21,14 +21,23 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: subosito/flutter-action@v1
+      - name: Set up Flutter
+        uses: subosito/flutter-action@v1
+        with:
+          flutter-version: 'stable'
+        
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)